jekyll/_cci2/creating-orbs.md

```diff
@@ -6,39 +6,36 @@ description: "Starting point for Creating CircleCI Orbs"
 categories: [getting-started]
 order: 1
 ---
-
+## Introduction
 Orbs are reusable packages of CircleCI configuration that you may share across projects, enabling you to create encapsulated, parameterized commands, jobs, and executors that can be [used across multiple projects]({{ site.baseurl }}/2.0/using-orbs/). 
 
 [Orbs]({{ site.baseurl }}/2.0/orb-intro/) are made available for use in a configuration through the `orbs` key in the top level of your 2.1 [.circleci/config.yml]({{ site.baseurl }}/2.0/configuration-reference/) file.
 
-## Orb Publishing Process
-
-To publish an orb, follow the steps listed below as an org Admin.
-
-1. Claim a namespace (assuming you don't yet have one), using the command `circleci namespace create <NAMESPACEID> <VCS> <YOURORG>`. For example:
-`circleci namespace create sandbox github CircleCI-Public`
+### Orb Publishing Process
 
-2. Create the orb inside your namespace, for example:
-`circleci orb create sandbox/hello-world`
+Before working with orbs, you may find it helpful to gain a high-level understanding the end-to-end orbs publishing process. The diagram shown below illustrates the orbs publishing process.
+![orbsworkflow-v2 image](  {{ site.baseurl }}/assets/img/docs/orbsworkflow-v2.png)
 
-3. Create the content of your orb in a file. You will generally perform this action in your code editor in a git repo made for your orb. For example, let's assume a file in `/tmp/orb.yml` could be made with a simple orb similar to:
-`echo '{version: "2.1", description: "a sample orb"}' > /tmp/orb.yml`
+#### Steps 1 - 3 - Use the CircleCI CLI
+The first step you must perform in the orbs publishing process is to download and install, update, and then configure the CircleCI CLI for use with CircleCI orbs. Detailed instructions for how to install, update and configure the CLI can be found in the [Get the new CircleCI CLI](https://circleci.com/docs/2.0/creating-orbs/#get-the-new-circleci-cli) section on this page.
 
-4. Validate that your code is a valid orb using the CLI. For example, using the path above you could use:
-`circleci orb validate /tmp/orb.yml`
+#### Step 4 - Validate your build configuration
```

Comment:
This step isn't really about validating your build configuration, it's about double-checking that you installed the CLI correctly.

Reply:
The title and content should reflect that.

Reply:
Separately, this step assumes that someone already has a configuration file available. The link assumes they are running the command from the root file of a repo that has `.circleci/config.yml` set up correctly.

They might not/it would be nice to offer an alternate config to check/some indication of how you'd get a config there.

Reply:
Renamed steps as requested.